### ISSUE TITLE
"lerna": "^4.0.0"

### ISSUE THREAD

**Issue by kostikovk** [atlassian/cz-lerna-changelog#25]
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: ui@undefined
npm ERR! Found: lerna@4.0.0
npm ERR! node_modules/lerna
npm ERR!   dev lerna@"^4.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer lerna@"^3.0.0" from cz-lerna-changelog@2.0.3
npm ERR! node_modules/cz-lerna-changelog
npm ERR!   dev cz-lerna-changelog@"*" from the root project
```

**Comment by leejunhui** [comment]
Same issue,it seems current version of this library was not prepared to adapt with the newest version of Lerna(4.x).

**Comment by hayesmaker** [comment]
until this is fixed - we get around by running `npm install --legacy-peer-deps` - but it would be nice if this PR got merged:
https://github.com/atlassian/cz-lerna-changelog/pull/26

**Comment by janbiasi** [comment]
Any updates? ...

**Comment by kostikovk** [comment]
> until this is fixed - we get around by running `npm install --legacy-peer-deps` - but it would be nice if this PR got merged: #26

Yeah, It works. I have used the same solution but it will be more comfortable don't use `--legacy-peer-deps`.
